src/mediatype-icon.ts

```diff
@@ -82,11 +68,16 @@ export class MediatypeIcon extends LitElement {
     }
 
     const displayMediatype = this.displayMediaType || '';
+    const fillColor = mediatypeColor[displayMediatype];
 
     return html`
-      <div id="icon" class="${this.showText ? 'show-text' : 'hide-text'}">
+      <div
+        id="icon"
+        class="${this.showText ? 'show-text' : 'hide-text'}"
+        style="--iconFillColor:${fillColor}"
```

Comment:
Suggestion: Use https://lit.dev/docs/templates/directives/#ifdefined, ifDefined directive to: 
- allow fillcolor to be undefined (I am not clear about the state of the `fillColor` when `displayMediatype` isn't a key match in `mediatypeColor`)
- remove style attribute